src/marqo/tensor_search/on_start_script.py

```diff
@@ -105,14 +105,16 @@ def __init__(self):
         pass 
 
     def run(self):
-        self.logger.info("downloading nltk data....")
+        
         import nltk
 
         try:
             nltk.data.find('tokenizers/punkt')
```

Comment:
What are we doing here? Also, we should remove `NLTK` from on_startup_script`

Reply:
it's just a check to confirm that nltk was properly installed at buildtime (the message log outputs confirmation). but i can remove it

Reply:
we'll keep it in, since it shouldn't cause much latency, and it gives us confirmation on startup that it worked properly

Reply:
yeah there shouldn't be any noticeable overhead checking if the file is here